src/steps/05-PerformanceRequirements/DOW/OtherOfferingSummary.vue

```diff
@@ -7,15 +7,18 @@
             Your {{ serviceGroupVerbiageInfo.headingSummary }}
           </h1>
           <p>
-            If you need more {{ serviceGroupVerbiageInfo.typeForText }}s, add them below. 
-            You can also edit or delete any info from the 
-            {{ serviceGroupVerbiageInfo.typeForText }}s that 
-            you have already entered. When you’re done, click "Continue" and we will 
-            move on to your 
+            If you have more requirements for this category, add them below. You can
```

Comment:
`{{ serviceGroupVerbiageInfo.typeForText }}` -- this needs to go back in here exactly as it was before. It did not belong at the end of the paragraph where you had it in your previous commit. I didn't request to change this line, but I see how it does have it as such in the AC. The AC is incorrect for the first part of this paragraph -- no changes were needed - it's only the last part of the last sentence that needed an update based on S or TS. I will leave a note in the comments section for the ticket.